ionoscloud/data_source_server.go

```diff
@@ -489,5 +494,19 @@ func dataSourceServerRead(ctx context.Context, d *schema.ResourceData, meta inte
 		return diag.FromErr(err)
 	}
 
+	// Labels logic
+	labelsResponse, apiResponse, err := client.LabelsApi.DatacentersServersLabelsGet(ctx, datacenterId.(string), d.Id()).Depth(1).Execute()
+	logApiRequestTime(apiResponse)
+	if err != nil {
+		return diag.FromErr(fmt.Errorf("error occured while fetching labels for server with ID: %s, datacenter ID: %s", d.Id(), datacenterId.(string)))
```

Comment:
missing err

Reply:
@cristiGuranIonos done in 586238e4619fe7d41d00334bfc01c629d834a759